src/main/java/uk/co/objectivity/test/db/beans/xml/Compare.java

```diff
@@ -56,12 +56,18 @@
     @XmlAttribute
     private boolean fileOutputOn = false;
 
+    @XmlAttribute
+    private String keyColumns = null;
+
     @XmlAttribute
     private boolean minusQueryIndicatorOn = false;
 
     @XmlElement(name = "sql")
     private List<Sql> sqls;
 
+    @XmlElement(name = "beforeSqls")
+    private List<BeforeSqls> beforeSqls;
+
     @XmlElement(name = "assert")
```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [Nm: Use of identifier that is a keyword in later versions of Java (NM_FUTURE_KEYWORD_USED_AS_IDENTIFIER)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#nm-use-of-identifier-that-is-a-keyword-in-later-versions-of-java-nm-future-keyword-used-as-identifier).
The identifier is a word that is reserved as a keyword in later versions of Java, and your code will need to be changed in order to compile it in later versions of Java.

Reply:
fixed, thank you